pinata/test/pinata_test.go

```diff
@@ -0,0 +1,48 @@
+package pinata
```

Comment:
By the way, let the test be in the same folder level as the package itself. 
Which means you don't need to create a new `test` folder.